remappings.txt

```diff
@@ -0,0 +1,2 @@
+ds-test/=lib/forge-std/lib/ds-test/src/
```

Comment:
I dont think we should be importing `ds-test` anywhere anymore now that we have `forge-std/Test.sol`